src/testRunner/unittests/tsc/composite.ts

```diff
@@ -0,0 +1,81 @@
+namespace ts {
+    describe("unittests:: tsc:: composite::", () => {
+        verifyTsc({
+            scenario: "composite",
+            subScenario: "when setting composite false on command line",
+            fs: () => loadProjectFromFiles({
+                "/src/project/src/main.ts": "export const x = 10;",
+                "/src/project/tsconfig.json": Utils.dedent`
+                    {
+                        "compilerOptions": {
+                            "target": "es5",
+                            "module": "commonjs",

```

Comment:
Is this supposed to also contain `"composite": true`, so it can demonstrate that the commandline `false`/`null` actually overrides the tsconfigs value?